src/lib/Containers/Conversation.tsx

```diff
@@ -87,12 +78,8 @@ export class Conversation extends React.Component<RelayProps, State> {
     // End of the list reached. Increase page size. Relay
     // will fetch only the required data to fill the new
     // page size.
-    if (item.key === this.state.messages.length - 1) {
-      this.props.relay.setVariables({ pageSize: this.state.messages.length + PAGE_SIZE }, readyState => {
-        if (readyState.done) {
-          debugger
-        }
-      })
+    if (item.key === this.props.me.conversation.messages.edges.length - 1) {
```

Comment:
Yea

This was just something to get the subsequent Relay call for more data (+ re-render), and make sure it's working.

I'll look into (or would love to pair) on the more proper scroll behavior. I'm also a bit confused with this 'reversed flat list' thingie so maybe I can pair with @sarahscott 

Either way this worked!